js/cards/cffa.ts

```diff
@@ -485,7 +486,7 @@ export default class CFFA implements Card, MassStorage<BlockFormat>, Restorable<
         }
         const { blocks, readOnly } = blockDisk;
         const { name } = blockDisk.metadata;
-        let ext;
+        let ext: '2mg' | 'po';
```

Comment:
The reason this is only `'2mg' | 'po'` is because those are the only actual values that `ext` will ever have. This is because the extension is not taken from the data in the CFFA card—it's only what is deduced by whether there is a `_metadata` entry for the drive. There may be a better way to do this, but I haven't spent any time trying to figure it out since I'm mainly concerned with the Disk II card at this point.

Reply:
Yeah, this needs to be split out into two parts, really - the extension of the file for downloading purposes, and the actual format of the bytes for processing purposes. Since the actual format is implied by the fact that this is a block device, `ext` should really represent the actual file extension.